src/lib/heatmap.ts

```diff
@@ -3,7 +3,7 @@ import {AMINO_ACIDS, AMINO_ACIDS_BY_HYDROPHILIA} from './amino-acids.js'
 /** Codes used in the right part of a MaveHGVS-pro string representing a single variation in a protein sequence. */
 const MAVE_HGVS_PRO_CHANGE_CODES = [
   {codes: {single: '='}}, // Synonymous AA variant
-  {codes: {single: '*'}}, // Stop codon
+  {codes: {single: '*', triple: 'TER'}}, // Stop codon
```

Comment:
It seems to me that technically, only the `=` matches the RegEx that validates simple protein variants. That seems fine though, since it seems like the mave-hgvs package wants to enforce the `TER` and `DEL` strings rather than the shorthand notation.

See: https://github.com/VariantEffect/mavedb-ui/blob/c1475cae81e2c7add1c049a10b0d8c26b7726f58/src/lib/mave-hgvs.ts#L19

Reply:
If we wanted to allow those single codes through, it would be pretty trivial to update the RegEx to something like `/^p\.([A-Za-z]{3})([0-9]+)([A-Za-z]{3}|=|\*|-)$/ `, but it doesn't seem like we necessarily want to support these shorthand codes.

Reply:
Thanks -- I updated this with a comment about allowing noncompliant MaveHGVS strings.